src/user/user.model.js

```diff
@@ -0,0 +1,79 @@
+import mongoose from "mongoose";
+
+const [ACTIVE, INACTIVE] = ["ACTIVE", "INACTIVE"];
+
+const UserSchema = mongoose.Schema({
```

Comment:
Es preferible que solo importes las props que se usaran

```suggestion
import { Schema, model } from "mongoose";

const [ACTIVE, INACTIVE] = ["ACTIVE", "INACTIVE"];

const UserSchema = Schema({/* props */});

export default model("User", UserSchema);
```